javascript/src/test/components/BasicCourseSearch/BasicCourseTable.test.js

```diff
@@ -4,15 +4,26 @@ import BasicCourseTable from "main/components/BasicCourseSearch/BasicCourseTable
 import * as courseFixtures from "main/fixtures/Courses/courseFixtures";
 import userEvent from "@testing-library/user-event";
 import { useAuth0 } from "@auth0/auth0-react";
+import useSWR from "swr";
 jest.mock("@auth0/auth0-react");
-
-describe("BasicCourseTable tests", () => {
-  beforeEach(() => {
-    useAuth0.mockReturnValue({
-      isAuthenticated: true,
+jest.mock("swr");
+
+  describe("BasicCourseTable tests", () => {
+    beforeEach(() => {
+      useAuth0.mockReturnValue({
+        user: {
+          name: "test user",
+          email: "test@test.com",
+          picture: "https://picsum.photos/200",
+        },
+      });
+      useSWR.mockReturnValue({
+        data: {
+          role : "Admin"
+        }
+      })
     });
-
-  })
+  
```

Comment:
A suggestion for a test (if we don't already have it): can we add a test that checks the visibility of this column? If you're not logged in or a guest, we check if it's not present, if you're a member or admin, we check if it's present.